keyboards/handwired/tkk/config.h

```diff
@@ -0,0 +1,28 @@
+// Copyright 2022 Joe Fass (@jfass)
```

Comment:
This file should be removed

Reply:
Still needs to be removed.

Reply:
Done! .. unless I need to do something to include that last commit in the PR?